webapp/channels/src/components/dynamic_virtualized_list/index.jsx

```diff
@@ -212,6 +212,8 @@ export class DynamicVirtualizedList extends PureComponent {
     }
 
     componentWillUnmount() {
+        cleanupSharedObserver();
```

Comment:
Could this component could mount and unmount several times in the app lifetime? If so, we should move this higher up to do it only once when the entire app exits.

Reply:
It might, but I wouldn't think any improvement from that would be worth the extra complexity to do that. The similar `AutoSizer` component made by the dev of the libraries that `DynamicVirtualizedList` is based off of similarly creates an observer on mount and then cleans it up on unmount